### DISCUSSION TITLE
qskscrollarea‘s problem

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#472] [Q&A] [answered]
When the sub-control of qskscrollarea rewrites the mouse wheel, it cannot receive it and is intercepted by qskscrollarea wheelEvent(QWheelEvent * event)

**Comment by hestrro** [comment]
I can receive via an event event

**Comment by uwerat** [comment, marked as answer]
Events are processed bottom up. Maybe you forgot QskItem::setWheelEnabled( true );
